examples/wallet/app/utils/nodeConnection.js

```diff
@@ -28,7 +36,25 @@ export function getAccountState(identifier: Identifier): Promise<AccountState> {
           Object.assign(acc, { [poolId]: { parts: amount } }),
         {}
       )
-    }));
+    }))
+    .catch(error => {
+      if (error && error.response && error.response.status === 404) {
+        const hardcoded = {
```

Comment:
As I understand, this are defaults for the cases when the account was never used in the network (maybe add little comment?). Personally, I think creating an object just to separate it in parts and reassemble it later it's a bit unnecessary. I would just assign `const balance: Balance = 0` and so on... But it's just a detail.